x-pack/plugins/security_solution/common/endpoint/service/host_isolation/utils.test.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isVersionSupported, isOsSupported, isIsolationSupported } from './utils';
+
+describe('Host Isolation utils isVersionSupported', () => {
```

Comment:
not necessary to take action:

Jest support table tests which makes these kind of repetitive tests easier to read and write. 

https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Example:

```
test.each`
  a           | b           | expected
  ${'8.14.9'} | ${'7.13.0'} | ${true}
  ${'8.14.9'} | ${'9.14.0'} | ${false}
`('should validate that version $a is equal($expected) to $b', ({ a, b, expected }) => {
  expect(
    isVersionSupported({
      currentVersion: a,
      minVersionRequired: b,
    })
  ).toEqual(expected);
});
```

Reply:
thanks for the tip, I didn't know that!  I've refactored my tests to reflect it.